moveit_setup_assistant/src/tools/moveit_config_data.cpp

```diff
@@ -529,6 +529,34 @@ bool MoveItConfigData::outputFakeControllersYAML(const std::string& file_path)
     emitter << YAML::EndSeq;
     emitter << YAML::EndMap;
   }
+
+  emitter << YAML::EndSeq;
+
+  // Add an initial pose for each group
+  emitter << YAML::Key << "initial" << YAML::Comment("Default robot poses can be defined here");
```

Comment:
I was actually thinking of something like this, if there was no pose defined at all
(otherwise you loose feature recovery):
```yaml
initial:
#  - group: panda_arm
#    pose: home
```
Also, I suggest the following wording:
```suggestion
  emitter << YAML::Key << "initial" << YAML::Comment("Define initial robot poses");
```

Reply:
Sounds good. I committed it like that.